docs/devops/snippets/DotNet.GitHubAction/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace DotNet.GitHubAction.Extensions
+{
+    static class ServiceCollectionExtensions
+    {
+        internal static IServiceCollection AddGitHubActionServices(
+            this IServiceCollection services)
+        {
+            // Omitted for brevity
+            // https://github.com/dotnet/samples/blob/008f9e197fb5d0ebb3a41216ee5fdde01869368f/github-actions/DotNet.GitHubAction/DotNet.GitHubAction/Extensions/ServiceCollectionExtensions.cs
```

Comment:
Instead of a commit hash, maybe just the `main` branch?

Reply:
I grabbed a permalink, but I'm fine with main -- I didn't know if we were supposed to use permalinks or not?

Reply:
I'd go with `main`. I don't know of a standard for this. I think we're fine with the branch on the assumption that any major updates to the code would also be reflected in updates in the article.